### ISSUE TITLE
Write a project around versioning and automating of database schemas

### ISSUE THREAD

**Issue by illicitonion** [CodeYourFuture/immersive-go-course#61]
Manually setting up databases on your machine for testing is annoying and fiddly.

Having to use a single shared database for testing servers is annoying and lacks isolation.

We should consider introducing checking in schemas and automating creation / migration.

**Comment by tgvashworth** [comment]
Yes I think good idea. I have migrations in the `buggy-app` project — could be improved for sure, in particular around testing.
